packages/design-system/src/components/toggle-group.tsx

```diff
@@ -30,11 +31,6 @@ export const Item = styled(ToggleGroupPrimitive.Item, {
   },
   "&:last-child": { borderTopRightRadius: 4, borderBottomRightRadius: 4 },
   "&:hover": { backgroundColor: "$slateA3" },
-  "&[data-state=on]": {
-    backgroundColor: "$slateA5",
-  },
-  "&:focus": {
-    position: "relative",
-    boxShadow: `0 0 0 2px black`,
-  },
+  "&:focus": { boxShadow: "0 0 0 $spacing$2 $colors$blue10", zIndex: 1 },
```

Comment:
was there again an issue with stacking context or are you fixing it preemptively? I wonder what we should standardized on, `1` feels random.

Maybe the issue was again somewhere else?

Reply:
please add that as a comment, super non-obvious